website/content/api-docs/system/config-group-policy-application.mdx

```diff
@@ -0,0 +1,78 @@
+---
+layout: api
+page_title: /sys/config/group-policy-application - HTTP API
+description: The '/sys/config/group-policy-application' endpoint is used to configure the global mode for group policy application.
+---
+
+# `/sys/config/group-policy-application`
+
+The `sys/config/group-policy-application` endpoint can be used to configure the
+mode of policy application for identity groups in Vault.
+Currently, it only supports two modes, `within_namespace_hierarchy` (default), which
+restricts policy application to only policies from groups from parent namespaces, and
+`any` which does not restrict policy application, and policies will apply from any namespace,
+irrespective of namespace hierarchy.
```

Comment:
I'm not sure this is sufficient for a typical Vault user to understand the implications.  Can we spell out in a little more detail what we mean by policy application?

Reply:
That's fair, I'll update it a little

Reply:
I tried to update it a bit, but I am finding it a bit tricky to find the words to explain this kind of thing both accurately and succinctly. If you think it's still unclear, let me know and I'll take another stab at it.

Reply:
How about something like:

```
Historically, Vault would allow you to add entities and groups from any namespace into an identity group.  However, any policies attached to that group would only apply when the vault token authorizing a request was created in a the same namespace as that group, or a descendent namespace. This endpoint allows relaxing that restriction: when the mode is set to the default, `within_namespace_hierarchy`, the historical behaviour is maintained, but when set to `any`, group policies apply to all members of a group, regardless of what namespace the request token came from.
```

Reply:
I modified it slightly, but thanks for this, this is perfect. I appreciate this a lot! Going to resolve this, but feel free to re-open or raise another if you think further language updates will help.